src/pages/Comments/index.jsx

```diff
@@ -1,41 +1,91 @@
-import { useEffect, useState } from "react";
+import { useEffect, useState } from 'react';
+import Comment from '../../components/comment';
 
 const Comments = () => {
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState(false);
   const [comments, setComments] = useState([]);
+  const [addedComment, setAddedComment] = useState("");
+  const [addCommentClicked, setAddCommentClicked] = useState(false);
+  const [username, setUsername] = useState("");
 
   useEffect(() => {
     fetchComments();
     // let's not forget to use state for the Promise values (data, loading, error)
-  }, [])
+  }, []);
 
   const fetchComments = async () => {
     try {
-      const response = await fetch("https://api.mocki.io/v2/a20ae30b/comments")
+      const response = await fetch('https://api.mocki.io/v2/a20ae30b/comments');
+      // const data = fetch("data.json")
       const data = await response.json();
-      setComments(data.comments)
-      setLoading(false)
-      console.log(comments)
+      setComments(data.comments);
+      setUsername(data.currentUser.username)
+      setLoading(false);
+    } catch (error) {
+      setLoading(false);
+      setError(true);
+    }
+  };
 
+  const handleAddCommentSubmit = (e) => {
+    e.preventDefault();
+    if ( addedComment === '') {
+      return;
     }
-    catch (error) {
-      setLoading(false)
-      setError(true)
+
+    const addedCommentData = {
+        "id": 5,
+        "content": addedComment,
+        "createdAt": "now",
+        "score": 0,
+        "user": {
+          "image": {
+            "png": "",
+            "webp": ""
+          },
+          "username": username,
+        },
+        "replies": []
     }
+  
+    let newComments = comments.concat(addedCommentData);
```

Comment:
this could be a const here i think? that might have been my fault

Reply:
updated